docs/intro.asciidoc

```diff
@@ -19,6 +19,8 @@ You will also need to {apm-server-ref}/rum.html[enable frontend endpoints in the
 [[getting-started]]
 == Getting started
 
+=== Using Package Managers
```

Comment:
I think you should add `[float]`, this will make it generated in the same page instead of a separate one. 
Also there should be page ID so we can refer to these sections individually, something like `[[using-package-managers]]`